openstack/loadbalancer/v2/flavors/requests.go

```diff
@@ -0,0 +1,111 @@
+package flavors
+
+import (
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+// LIST
+
+type ListOptsBuilder interface {
+	ToFlavorListQuery() (string, error)
+}
+
+type ListOpts struct {
+	Fields []string `q:"fields"`
+}
+
+func (opts ListOpts) ToFlavorListQuery() (string, error) {
+	q, err := gophercloud.BuildQueryString(opts)
+	return q.String(), err
+}
+
+func List(c *gophercloud.ServiceClient, opts ListOptsBuilder) pagination.Pager {
+	url := rootURL(c)
+	if opts != nil {
+		query, err := opts.ToFlavorListQuery()
+		if err != nil {
+			return pagination.Pager{Err: err}
+		}
+		url += query
+	}
+	return pagination.NewPager(c, url, func(r pagination.PageResult) pagination.Page {
+		return FlavorPage{pagination.LinkedPageBase{PageResult: r}}
+	})
+}
+
+// CREATE
+
+type CreateOptsBuilder interface {
+	ToFlavorCreateMap() (map[string]interface{}, error)
+}
+
+type CreateOpts struct {
+	Name            string `json:"name,omitempty"`
```

Comment:
Name is a required parameter, at least according to the docs.
```suggestion
	Name            string `json:"name" required:"true"`
```